builtin/settingtypes.txt

```diff
@@ -2127,6 +2115,20 @@ show_technical_names (Show technical names) bool false
 #    Changing this setting requires a restart.
 enable_sound (Sound) bool true
```

Comment:
Should be moved to `[*Audio]`. (Ironically, this is the only non-tmp setting of the three.)

Reply:
No, this is intentionally not visible in the GUI as it's redundant with mute sound

Reply:
Oh, so the ingame audio menu shows exactly this section? Didn't know.

Reply:
Ah, so `[*Temporary Settings]` is actually `[*Hidden Settings]`. I see.